webapp/src/components/sidebar_right/github_items.jsx

```diff
@@ -103,14 +104,37 @@ function GithubItems(props) {
             }
         }
 
+        let hasConflict = '';
+        if (item.mergeable != null && !item.mergeable) {
```

Comment:
Is there a better syntax for this check?

Reply:
A change on the variable name (and assignment logic) to "item.mustMerge"?

Reply:
What makes `mustMerge` a better variable name then `mergeable'?

Reply:
It just makes the check simpler. Bet it null or false, you can skip creating the conflict icon by just checking `if (item.mustMerge) {`

Reply:
I not sure I get your suggestion. Do you mean renaming `PRDetails.Mergeable` to `PRDetails.MustMerge` in `server` and `webapp`?

Reply:
Yes. Starting from `PRDetails.MustMerge := !prInfo.GetMergeable()` all the way to the front-end. It will simplify this check and does not introduce any more code than a negation. Semantic/readability-wise is pretty similar too.

Reply:
Hm, I see. `MustMerge` is such a clear name to me. Maybe `hasConflict`?

Reply:
LGTM

Reply:
After thinking a bit more about this, I'm wondering if the API provided by the plugin should be similar to the on from GitHub. Hence, `Mergeable` would be the correct name, given that it's also the name GitHub uses in there API. Let me know what you think.

Reply:
+1 on `Mergeable` being the correct name here.